examples/api/java/Relations.java

```diff
@@ -0,0 +1,215 @@
+/*********************                                                        */
+/*! \file Relations.java
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Mudathir Mahgoub
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Reasoning about relations with CVC4 via Java API.
+ **
+ ** A simple demonstration of reasoning about strings with CVC4 via Jave API.
+ **/
+
+import edu.nyu.acsys.CVC4.*;
```

Comment:
I think this should say "stanford" not "nyu".

Reply:
I remembered it was mentioned in the meeting, but my version was a bit behind master. 
It is fixed to `edu.standfaord.CVC4.*` What about the other name called  `uiowa` :)